app/scenes/Search/Search.js

```diff
@@ -319,6 +329,18 @@ class Search extends React.Component<Props> {
               mode={ArrowKeyNavigation.mode.VERTICAL}
               defaultActiveChildIndex={0}
             >
+              {collectionResults.map((result) => {
+                const collection = collections.data.get(result.collection.id);
```

Comment:
No need to get the collection from the store again, we already have it in `collectionResults`